quickscreen/summary/summary_stats.py

```diff
@@ -23,7 +23,14 @@ def missing_summary(df, type="columns"):
     --------
     >>> missing_summary(pd.DataFrame(df), "rows")
     """
-    data = Missing(df, type)
+    try:
+        data = Missing(df, type)
+        if not hasattr(data, "df"):
+            raise Exception
+    except:
+        print("Can not complete function")
```

Comment:
maybe include in the print statement why it cannot complete the function?

Reply:
When the function tries to use the class, the class raises the exception and prints the reason for the error so the output is is two line print statement with this line below the class error output line.